src/backend/distributed/deparser/deparse_function_stmts.c

```diff
@@ -789,7 +789,14 @@ static void
 AppendGrantOnFunctionFunctions(StringInfo buf, GrantStmt *stmt)
 {
 	ListCell *cell = NULL;
-	appendStringInfo(buf, " ON %s ", ObjectTypeToKeyword(stmt->objtype));
```

Comment:
ObjectTypeToKeyword seems to be used in many other places.

1) Should we change ObjectTypeToKeyword instead?
2) Do we need anything to support those code-paths with aggragates?

Reply:
1. objecttypetokeyword definition is correct. We do not need to use it for GRANT/REVOKE statements which can use [ROUTINE](https://www.postgresql.org/docs/current/sql-grant.html) keyword to support functions, procedures and aggregates.
2. Required changes are done to support GRANT/REVOKE for aggregates.